src/test/libcephfs/test.cc

```diff
@@ -655,6 +671,24 @@ TEST(LibCephFS, Xattrs_ll) {
   ceph_shutdown(cmount);
 }
 
+/**
+ * Test if sending dummy op crashes MDS or not.
+ */
+TEST(LibCephFS, check_dummy_op) {
+  struct ceph_mount_info *cmount;
+  ASSERT_EQ(ceph_create(&cmount, NULL), 0);
+  ASSERT_EQ(ceph_conf_read_file(cmount, NULL), 0);
+  ASSERT_EQ(0, ceph_conf_parse_env(cmount, NULL));
+  ASSERT_EQ(ceph_mount(cmount, NULL), 0);
+
+  UserPerm *perms = ceph_mount_perms(cmount);
+
+  int res = ceph_check_dummy_op(cmount, perms);
+  ASSERT_NE(res, 0);
+
+  ceph_shutdown(cmount);
+}
```

Comment:
This test belongs in src/test/client/ops.cc (a new file).

Reply:
I've moved it. PTAL.